### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by atomiix**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | For security reasons, some dependencies should be updated, this PR aims to do so.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | CI should be green
| Possible impacts? |

### CODE REVIEW

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
@atomiix PR approved. I wait for you to come back before we merge, because I'm not 100% sure production environment can use composer 2.2

**Review by matthieu-rolland — APPROVED** [review]

**Pull request merged**
